src/subscriptions.js

```diff
@@ -41,7 +41,7 @@ export default tracker => ({ dispatch }) => next => action => {
       stopSubscription(action);
     };
 
-    runContextual(stop);
+    run(stop);
```

Comment:
[above comment](https://github.com/samybob1/meteor-redux-middlewares/pull/15/files#r94862359)

could just be `setTimeout`